boards/arm/nucleo_h743zi/doc/index.rst

```diff
@@ -123,6 +123,8 @@ features:
 +-------------+------------+-------------------------------------+
 | WATCHDOG    | on-chip    | independent watchdog                |
 +-------------+------------+-------------------------------------+
+| USB         | on-chip    | usb_device                          |
```

Comment:
```suggestion
| USB OTG FS  | on-chip    | USB device                          |
```

Reply:
I don't know that it makes sense to make this distinction here. "USB" versus "USB OTG FS". We don't make that level of distinction elsewhere. The "usb_device" is very specific, it matches our terminology elsewhere as well.